lib/stat_generator.rb

```diff
@@ -48,19 +48,74 @@ def total_ties
       game.away_goals == game.home_goals
     end
   end
-
+  
   def percentage_ties
     (total_ties / count_of_games.to_f).round(2)
+  end 
+  
+  def seasons
+    @games.map do |game|
+      game.season
+    end.uniq
+  end
+  
```

Comment:
I like this seasons method. I think it is more efficient than the version I wrote